sig-testing/README.md

```diff
@@ -33,20 +33,26 @@ The Chairs of the SIG run operations and processes governing the SIG.
 ## Subprojects
 
 The following subprojects are owned by sig-testing:
-- **test-infra**
+- **kind**
+  - Description: Kubernetes IN Docker. Run local Kubernetes clusters using Docker containers as nodes.
```

Comment:
not sure if redundant, but maybe something about how this is for testing "local Kubernetes test clusters"? /shrug